hr_holidays_hour/models/hr_holidays_status.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Onestein (<http://www.onestein.eu>)
+# Copyright 2019 Coop IT Easy SCRLfs
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class HrHolidaysStatus(models.Model):
+    _inherit = "hr.holidays.status"
+
+    @api.multi
+    def get_hours(self, employee):
+        self.ensure_one()
+        result = {
+            "max_hours": 0,
+            "remaining_hours": 0,
+            "hours_taken": 0,
+            "virtual_remaining_hours": 0,
+        }
```

Comment:
I'd use variables in the code and build the dictionary at the end of the method.